Khan/math-input
# FEI-4197: Update react-redux to 7.2.6

## Summary:
FEI-4163 is updating webapp to use react-redux 7.2.6 so that we can use the hooks that are available in that future.  In order to ensure that math-input will continue to work after this upgrade we also need to update react-redux here as well.  One of the changes that was made in react-redux was for connect() to return a functional component.  There are a few places where we use refs with connect() wrapped components.  Thankfully, in the new version of react-redux, connect() has an option for it to use forwardRef() so that we can use refs with the components it returns.  I've updated all calls of connect() to use this option so we don't have to worry about tracking down which places it's actually needed.

Issue: FEI-4197

## Test plan:
- yarn start
- load http://localhost:9000/ in mobile/responsive mode in your web browser
- tap on the input field, see the keypad show up at the bottom
- type some keys, see the numbers/symbols appear in the input
- click on the input again, drag the cursor handle
- tap the dismiss button on the keypad, see the keypad disappear
